### PR TITLE
fix: the action change modal is not shown when the action is first se…

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/tasks/15698313

**Changes** 🏗

* Not showing action change modal when action is changed for first time and title is populated

Resolves [#1586](https://github.com/JoinColony/colonyCDapp/issues/1586)

### CODE REVIEW

File: src/components/v5/common/ActionSidebar/ActionTypeSelect.tsx [hunk 1]
```diff
@@ -94,7 +94,10 @@ const ActionTypeSelect: FC<ActionTypeSelectProps> = ({ className }) => {
                     return;
                   }
 
-                  if (Object.keys(formState.dirtyFields).length > 0) {
+                  if (
+                    Object.keys(formState.dirtyFields).length > 0 &&
+                    !!actionType
```
Comment (TomaszChmielPagepro) [inline src/components/v5/common/ActionSidebar/ActionTypeSelect.tsx:99]: Is  `!!` needed here?

Comment (CzarekDryl) [reply src/components/v5/common/ActionSidebar/ActionTypeSelect.tsx:99]: It wasn't needed

**Review by TomaszChmielPagepro — APPROVED** [review]

**Review by krawczykkatarzyna — APPROVED** [review]

**Review by jakubcolony — APPROVED** [review]
LGTM

**Review by ArmandoGraterol — APPROVED** [review]
Nice little UX improvement 👌

**Pull request merged**